trailblazer/services/tower/tower_api_service.py

```diff
@@ -17,8 +17,7 @@ def __init__(self, client: TowerAPIClient, store: Store) -> None:
     @handle_errors
     def update_jobs(self, analysis_id: int) -> list[Job]:
         analysis: Analysis = self.store.get_analysis_with_id(analysis_id)
-        workflow_id: str = get_tower_workflow_id(analysis)
-        response: TowerTasksResponse = self.client.get_tasks(workflow_id)
+        response: TowerTasksResponse = self.client.get_tasks(analysis.tower_workflow_id)
```

Comment:
Are the values in the column even populated yet? Seems like nothing has a tower_workflow_id yet?

Reply:
They are populated, just not in stage :)

Reply:
I mean, I am checking in production

Reply:
![Uploading image.png…]()

Reply:
Weird! Looking into it. I'm guessing those analyses have not been run via the tower API which is why they have not been assigned a workflow id

Reply:
Ah, I have forgotten to deploy the trailblazer web API which meant that the workflow id passed from cg was just dropped and never stored 😅

Reply:
They do seem to be populating now.